tests/static-check/run_checks.sh

```diff
@@ -8,7 +8,7 @@ function check_with_gcc() {
   rm -f config.cache
   make clean
   ./configure -C --enable-debug CC=gcc
-  local gcc_exceptions="-Wno-sign-compare -Wno-implicit-fallthrough -Wno-cast-function-type -Wno-type-limits"
+  local gcc_exceptions="-Wno-sign-compare -Wno-implicit-fallthrough -Wno-type-limits"
```

Comment:
Keep in mind that there will be a conflict now, since you are editing this line in 2 different PRs.